### ISSUE TITLE
Bug [Documentation] - Install instructions: MariaDB config bug

### ISSUE THREAD

**Issue by JedMeister** [znuny/manual#3]
### Environment

- OS:           Linux (Debian 11/Bullseye)
- Browser:      N/A
- Znuny version: 6.0+

### Expected behaviour

Install instructions give correct MariaDB config.

### Actual behaviour

[Suggested MariaDB config](https://doc.znuny.org/manual/releases/installupdate/install.html#database-configuration) is buggy and causes issues with `mysqladmin` (and possibly others?).

### How to reproduce

Add MariaDB config as suggested:
```
cat > /etc/mysql/mariadb.conf.d/50-znuny_config.cnf <<EOF
[client]
max_allowed_packet=256M

[mysqld]
innodb_file_per_table
innodb_log_file_size = 256M
max_allowed_packet=256M
EOF
systemctl restart mariadb
```
All seems well, until you try to run `mysqladmin`, e.g. to create a new DB:
```
root@otrs /# mysqladmin create newdb
mysqladmin: unknown variable 'max_allowed_packet=256M'
root@otrs /# echo $?
7
```
A bit of googling lead me to this "not a bug" bug report: https://bugs.mysql.com/bug.php?id=68313

As noted there:

> You can only put things in the [client] section that are supported by all clients. Typically you should just put things like user, port, and socket in there.

So the fix is to instead put it in the `[mysql]` section. Then the `mysql` client will use it, but `mysqladmin` (and perhaps other clients that don't support that option) won't choke on it. I.e.:

```
--- a/etc/mysql/mariadb.conf.d/50-znuny_config.cnf.orig
+++ b/etc/mysql/mariadb.conf.d/50-znuny_config.cnf
@@ -1,4 +1,4 @@
-[client]
+[mysql]
 max_allowed_packet=256M
 
 [mysqld]
```
After making the change:
```
root@otrs /# mysqladmin create newdb
root@otrs /# echo $?
0
```

### Additional information

Apologies if this is the wrong place to post this bug report. Also, I was going to open a pull request with the suggested change as well, but I'm not sure where the doc source is?

**Comment by JedMeister** [comment]
Thanks for moving this. Sorry that I missed this repo.

**Comment by rkaldung** [comment]
@JedMeister Thank you for reporting. You are 100% correct. We added this globally to have the setting valid for the mysql client and mysqldump. We will update the instructions.

**Comment by rkaldung** [comment]
@JedMeister It's fixed for [Znuny](https://github.com/znuny/manual/commit/ca494d5d4a13c7f4f651bf209e310e6bc264afe9) and [Znuny LTS](https://github.com/znuny/manual/commit/0f36552ac7ecff6b6d150299468235eedbcafedb)

**Comment by JedMeister** [comment]
Thanks @rkaldung but are you sure about this?:
```
[mysqldump]
max_allowed_packet=256M
```

My understanding was that `mysqldump` doesn't understand/accept the `max_allowed_packet` setting?! Unfortunately I don't have a server handy right now to test, but that was my understanding of the error message I hit. Moving it out of the `[client]` config to `[mysql]` means that the `mysql` CLI client will successfully use it, but it won't be forced on other clients (such as `mysqldump`) that don't understand it.

**Comment by rkaldung** [comment]
@JedMeister It does. It's mysqladmin that does not know/accept this setting. There could be one section for every client application in a configuration file. We need this setting because otherwise, dumping (=backing up) the database could fail because we accept packages up to 256M but can't output these large ones.
Anyway, I verified it:
<img width="883" alt="image" src="https://user-images.githubusercontent.com/378324/232391103-016a8b01-660b-4941-9a98-dd438214bfbc.png">

mysqladmin 'fixes' the pretty low setting to the minimum of 4k.

I checked MySQL 8 and MariaDB 10.5.
